server/test/unit/server/webservices/views/test_plugins.py

```diff
@@ -17,7 +18,53 @@ class TestDistributorsView(unittest.TestCase):
 
 
 class TestImporterResourceView(unittest.TestCase):
-    pass
+    """
+    Tests for the views of a single importer.
+    """
+
+    @mock.patch('pulp.server.webservices.controllers.decorators._verify_auth',
+                new=assert_auth_READ())
+    @mock.patch('pulp.server.webservices.views.plugins.generate_json_response')
+    @mock.patch('pulp.server.webservices.views.plugins.factory')
+    def test_get_importer_resource_view_existing_importer(self, mock_factory, mock_serializer):
+        """
+        Importer Resource should return a serialized object of the specified type
+        """
+        mock_manager = mock.MagicMock()
+        mock_manager.importers.return_value = [{'id': 'mock_importer_1'}, {'id': 'mock_importer_2'}]
+        mock_factory.plugin_manager.return_value = mock_manager
+        request = mock.MagicMock()
+        request.get_full_path.return_value = '/mock/path/'
+
+        importer_resource = ImporterResourceView()
+        importer_resource.get(request, 'mock_importer_2')
```

Comment:
and as before, it would be helpful to assert that this actually returns something sane.